packages/opencensus-core/src/common/version.ts

```diff
@@ -0,0 +1,19 @@
+
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const pjson = require('../../package.json');
```

Comment:
Let's use the [`pjson`](https://www.npmjs.com/package/pjson) module which I believe recursively descends directories until it hits a package.json file. That way we don't need to copy it into the `build` directory.

Reply:
My only concern with that is that `pjson` uses the `fs` node library, which means that `core` would be node-only and wouldn't work in (for example) a browser context. Is that okay? My understanding architecturally was that `core` should be environment agnostic, with other packages (`opencensus-nodejs`) that are environment specific.

If that's _not_ true, then yeah, something like pjson is a nice fit for this use case for sure and I could easily add that in.

Reply:
You're right. In that case, to avoid the file copy would it be possible to use a try-catch to descend either 2 or 3 directories, or hardcode `'../../../package.json'` instead? (I would prefer the former, but either works)